mypy/report.py

```diff
@@ -123,6 +123,8 @@ def should_skip_path(path: str) -> bool:
         return True
     if 'stubs' in path.split('/') or 'stubs' in path.split(os.sep):
         return True
+    if 'lib-stub' in path.split('/') or 'lib-stub' in path.split(os.sep):
+        return True
     return False
```

Comment:
I guess, that on a normal setup (both paths are on the same drive), the relative path would start with `...`, which leads to a True in line 123. Due to my workaround the path now absolute, so it would not work anymore. However is it semantically wrong to ignore `lib-stub`? What do you think?

Reply:
Skipping `lib-stubs` is probably fine.